drivers/entropy/entropy_stm32.c

```diff
@@ -103,6 +103,9 @@ static void configure_rng(void)
 {
 	RNG_TypeDef *rng = entropy_stm32_rng_data.rng;
 
+#ifdef STM32_CONDRST_SUPPORT
```

Comment:
How is this flag defined?

Reply:
the flag is defined by the presence of bit RNG_CR_CONDRST in the RNG CR of the peripheral

Reply:
right, I've missed it. Txs